client-cli/src/command/wallet_command.rs

```diff
@@ -31,6 +35,35 @@ pub enum WalletCommand {
         )]
         wallet_type: WalletKind,
     },
+    #[structopt(name = "export", about = "Backup wallet to a file")]
+    Export {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: String,
+        #[structopt(
+            name = "file",
+            short = "f",
+            long = "file",
+            parse(from_os_str),
+            help = "file to dump wallet"
+        )]
+        file: PathBuf,
```

Comment:
I suggest to treat a special value "-" as stdout and make it the default value, so the command can work with pipes in shell.